pkg/beegfs/controllerserver.go

```diff
@@ -378,7 +379,17 @@ func (cs *controllerServer) ListSnapshots(ctx context.Context, req *csi.ListSnap
 }
 
 func (cs *controllerServer) ControllerExpandVolume(ctx context.Context, req *csi.ControllerExpandVolumeRequest) (*csi.ControllerExpandVolumeResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "")
+	volumeID := req.GetVolumeId()
+	if len(volumeID) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "Volume ID not provided")
+	}
```

Comment:
Not sure if we still run CSI sanity tests. If we do and they are up-to-date, I'd be a bit surprised if they passed with this minimum number of checks. I would guess (though again, I really don't know this) that they'd try to throw things at us like a capacity that's smaller than the current capacity. (Maybe not though, as the spec might REQUIRE the container orchestrator not to do that.) Just something to check if you have time.

Reply:
We do still run CSI sanity tests and they are all [passing](https://github.com/ThinkParQ/beegfs-csi-driver/actions/runs/9822919929/job/27120167696#step:7:312) with this change, but you bring up a good point they are not up-to-date. I opened https://github.com/ThinkParQ/beegfs-csi-driver/issues/31 to get those updated at some point.